src/main/kotlin/model/SnowballRException.kt

```diff
@@ -58,4 +61,41 @@ sealed class SnowballRException(
             criterionId: String,
         ) : EntityNotPersistedException("Criterion", criterionId)
     }
+
+    /**
+     * Represents a specific type of exception that occurs when an entity is accessed by
+     * the current user, but they are not authorized.
+     *
+     * This exception is intended to provide a clear and structured way to handle
+     * scenarios where a particular entity is accessed, but the current user is not
+     * authorized to do so.
+     * It serves as a base class to define more specific "unauthorized" exceptions for
+     * various entities.
+     *
+     * @constructor Creates an [UnauthorizedException] with the current user's ID, the
+     * name and ID of the accessed entity.
```

Comment:
```suggestion
     * type and ID of the accessed entity.
```
Is it really the name of the entity? Could be confusing, because the name of a project is actually something different, right?

Reply:
Yes, I agree. I changed for all exceptions.